Cubical/Algebra/CommAlgebra/FGIdeal.agda

```diff
@@ -0,0 +1,22 @@
+{-# OPTIONS --cubical --safe --no-import-sorts #-}
```

Comment:
You don't need all of these flags anymore

Reply:
Should we change CONTRIBUTING.md then? It says all files should start like that.

Reply:
Ah, good point! Can you please update that as well?

Reply:
Never mind, I updated the contributing file in https://github.com/agda/cubical/pull/584

So just remove the `--cubical` and `--no-import-sorts` flags and this will be ready to be merged.

Reply:
Done